app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: [:edit, :update, :destroy]
+
+  def create
+    authorize! :create, Comment
+
+    @comment = Comment.new(comment_params)
+
+    if @comment.save
```

Comment:
This is what responders gem is for, https://github.com/plataformatec/responders

But somehow I can't really go for it, because I'd like to keep it explicit. And a controller should be slim and CRUD only methods anyway. So in my opinion, every controller should basically look like a plain Rails generated controller.

And there is "Lazy" Lookup for controllers since Rails 4.x which is where the flash messages themselves should live. http://guides.rubyonrails.org/i18n.html#looking-up-translations